Sources/Formworks/Design System/FWStyle.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  FWStyle.swift
+//  
+//
+//  Created by Cassia Aparecida Barbosa on 29/10/20.
+//
+
+import Foundation
+
+/// The form's color style.
+public enum FWStyle {
+	/// The default light color style of a form.
+	case light
+	
+	/// The default dark color style of a form.
+	case dark
```

Comment:
Nice to know that both modes are in theory implemented. Due to this, we need to put back on the style documentation the dark mode color codes.

Reply:
We still need to add some more colors since we've added new customisation options.